src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -83,9 +95,67 @@
     }
 @end
 
+@private clientStreamingMethodSampleCode(apiMethod)
+    $requests = [$request];
+    
+    // Write data to server and wait for a response
+    $streamingResponse = {@methodCallSampleCode(apiMethod)};
+    $result = $streamingResponse->writeAllAndWait($requests);
```

Comment:
Oh and as for your first statement - I suggested `pullStream` because the API method is `pull`. For the Speech API, the corresponding name would be `recognizeStream`.